TutorialApplication/OrganizationInfoViewController.swift

```diff
@@ -14,6 +14,87 @@ import UIKit
     
     static let kEmployeesOrderHasChanged = "EmployeesOrderHasChanged"
     
+    static let kOrganizationHasChanged = "OrganizationHasChanged"
+    
+    @IBAction func fetchOrganizations()
+    {
+        RequestManager.fetchOrganizations { (response) in
+        
+        var newOrganization:FFOrganization
+        
+        var newEmployee:FFEmployee
+            
+        let countOfOrganizations = DatabaseController.requestResults(for: nil, sortDescriptors: nil, entity: "FFOrganization")
```

Comment:
Before you parse new organizations, you should remove all previous ones including employees.